README.md

```diff
@@ -1,24 +1,90 @@
-# README
+![](https://img.shields.io/badge/Maen-AL--Khraisha-blue)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Budget-App
 
-Things you may want to cover:
+> This is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-* Ruby version
+## Live demo
+[Live Demo Link](https://budget-app-maen.herokuapp.com/)
 
-* System dependencies
+[Live Video Link](https://www.loom.com/share/ea6b6118e4ee496e89c6cb8cc6f40ab5)
 
```

Comment:
- [ ] Your video should contain a highlight about a piece of the code you find interesting or you are proud of.